.tekton/osc-monitor-pull-request.yaml

```diff
@@ -28,6 +28,8 @@ spec:
     value: 5d
   - name: dockerfile
     value: Dockerfile.monitor
+  - name: prefetch-input
```

Comment:
Suggested changes to the commit message:

1. This is because _it_ requires `yq`: I believe that _it_ refers to "checking go version". If so, I suggest rewording as "This is because version check requires yq".
2. Add the comment found in the sandboxed-containers-operator repository commits, see below:

```
Following Konflux doc:
- https://konflux-ci.dev/docs/building/prefetching-dependencies/#enabling-prefetch-builds-for-rpm
- https://konflux-ci.dev/docs/building/activation-keys-subscription/#configuring-an-rpm-lockfile-for-hermetic-builds
```

Reply:
Done for both rephrasing the commit message, and adding the doc link